### PR TITLE
Update Simple-Chinese Translation to latest.

### PR DESCRIPTION
**Pull request by zkdpower**
Update Simple-Chinese Translation to latest.

### CODE REVIEW

File: src/duckstation-qt/translations/duckstation-qt_zh-cn.ts [hunk 1]
```diff
@@ -7878,6 +7926,16 @@ Please configure a supported controller from the list above.</source>
         <source>Failed to open disc image &apos;%s&apos;: %s.</source>
         <translation>无法打开光盘镜像&apos;%s&apos;: %s。</translation>
     </message>
+    <message>
+        <location filename="../../core/system.cpp" line="2180"/>
+        <source>Failed to switch to subimage %u in &apos;%s&apos;: %s.</source>
+        <translation>无法在&apos;%s&apos;: %s切换到子镜像%u。</translation>
```
Comment (stenzek) [inline src/duckstation-qt/translations/duckstation-qt_zh-cn.ts:7932]: This will cause a crash, the core strings can't have the operand order changed.

Comment (zkdpower) [reply src/duckstation-qt/translations/duckstation-qt_zh-cn.ts:7932]: ok,i fix.

Comment (zkdpower) [reply src/duckstation-qt/translations/duckstation-qt_zh-cn.ts:7932]: "in" needs remain?

Comment (zkdpower) [reply src/duckstation-qt/translations/duckstation-qt_zh-cn.ts:7932]: i guess the sequence can't changed.

**Pull request closed without merging**